### ISSUE TITLE
dependency on libGL (menpo3d -> CyRasterize)

### ISSUE THREAD

**Issue by jabooth** [menpo/landmarkerio-server#12]
Just trying to get landmarkerio-server running on a fresh VM of 14.04 and we hit an error after 
```
conda install -c menpo/channel/master landmarkerio
```
```
Traceback (most recent call last):
  File "/home/jab08/miniconda/envs/lmio/bin/lmio", line 6, in <module>
    from landmarkerio.cache import cache_assets, filepath_as_asset_id_under_dir
  File "/home/jab08/miniconda/envs/lmio/lib/python2.7/site-packages/landmarkerio/cache.py", line 13, in <module>
    import menpo3d
  File "/home/jab08/miniconda/envs/lmio/lib/python2.7/site-packages/menpo3d/__init__.py", line 2, in <module>
    from . import rasterize
  File "/home/jab08/miniconda/envs/lmio/lib/python2.7/site-packages/menpo3d/rasterize/__init__.py", line 1, in <module>
    from .opengl import GLRasterizer
  File "/home/jab08/miniconda/envs/lmio/lib/python2.7/site-packages/menpo3d/rasterize/opengl.py", line 2, in <module>
    from cyrasterize.base import CyRasterizerBase
  File "/home/jab08/miniconda/envs/lmio/lib/python2.7/site-packages/cyrasterize/__init__.py", line 1, in <module>
    from cyrasterize.base import CyRasterizer
  File "/home/jab08/miniconda/envs/lmio/lib/python2.7/site-packages/cyrasterize/base.py", line 3, in <module>
    from cyrasterize.glrasterizer import GLRasterizer
ImportError: libGL.so.1: cannot open shared object file: No such file or directory
```

**Comment by jabooth** [comment]
Problem is the server I'm using is headless -> no X11. 
```
apt-get install libgl1-mesa-dev
```
needed to fix, but this drags in X11 that we really don't need for serving landmarks...

**Comment by patricksnape** [comment]
Just fix the import chain

**Comment by jabooth** [comment]
Fixed here: https://github.com/menpo/cyrasterize/commit/6fa28a2959c834c4da0e2e22a2a7c57b48402075 and released cyrasterize 0.2.2.

Menpo3d 0.1.0a3 released to depend on this and include a handful of other bugs that were present.
